test/api_container_logs_test.go

```diff
@@ -121,12 +124,13 @@ func (suite *APIContainerLogsSuite) TestSinceAndUntil(c *check.C) {
 	})
 
 	c.Assert(len(allLogs), check.Equals, 1)
+
+	DelContainerForceOk(c, name)
 }
 
 func (suite *APIContainerLogsSuite) TestCheckLeakByFollowing(c *check.C) {
 	name := "logs_check_leak_by_following"
 	command.PouchRun("run", "-d", "--name", name, busyboxImage, "sh", "-c", "while true; do sleep 1; done").Assert(c, icmd.Success)
```

Comment:
Maybe we should separate it into `create` and `start` two command. 

Since the `run` does a lot of things, `create`, `attach` and `start`, we cannot identify which action fails. 

In order to make the `defer` works well, I will update the case to make it robust.

Reply:
If no related flags are specified, I do not think `attach` will be involved. @fuweid 
Am I correct?

Reply:
Without any flags, the `run` doesn't call `attach` API. However, the `start` is always called after `create`. Since we doesn't test the `run` command here, I think we can separate it to make it stable. WDYT? I will update my cases after this PR.